daemon/loop.go

```diff
@@ -150,9 +150,7 @@ func (d *LoopVars) AskForImagePoll() {
 	}
 }
 
-// -- extra bits the loop needs
```

Comment:
no.  I was absolutely going to confirm this with you before suggesting we merge this change, by the way, but I tried to understand what it meant and decided it was added accidentally.  if you look at the diff https://github.com/weaveworks/flux/commit/f67ec554adc06624ebec09da2428d70930cd86e7 it looks like it may have been there to be a placeholder when things were moved around.

Again, I would have confirmed with you first before suggesting we merge this change, but it doesn't add any context and the extra line between it and the next function (which, by the way is 300 lines long and can hardly be considered a "bit") and the two that follow.